Jenkinsfile

```diff
@@ -141,60 +140,6 @@ def doBuildCocoa() {
   }
 }
 
-def doBuildDotNetOsx() {
-  return {
-    node('osx') {
-      getArchive()
-
-      try {
-        withEnv([
-          'PATH=$PATH:/usr/local/bin',
-          'DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer',
-          'REALM_ENABLE_ENCRYPTION=yes',
-          'REALM_ENABLE_ASSERTIONS=yes',
-          'MAKEFLAGS=\'CFLAGS_DEBUG=-Oz\'',
-          'UNITTEST_SHUFFLE=1',
-          'UNITTEST_REANDOM_SEED=random',
-          'UNITTEST_XML=1',
-          'UNITTEST_THREADS=1'
-        ]) {
-            sh '''
-              dir=$(pwd)
-              sh build.sh config $dir/install
-              sh build.sh build-dotnet-cocoa
```

Comment:
What will .NET use instead? Previously they were separate from Cocoa as the libraries they use are built differently (no bitcode for .NET as their toolchain can't handle it, fewer platforms for .NET). Did this requirement change?